README.md

```diff
@@ -60,6 +60,8 @@ jobs:
     - name: Start MongoDB
       uses: supercharge/mongodb-github-action@1.11.0
       with:
+        # Here we are using an image from Amazon's ECR rather than the default image from Docker Hub
+        mongodb-image: 'public.ecr.aws/docker/library/mongo'

```

Comment:
Please make the option to use a custom MongoDB docker image a new section  in the README.md. Please create a section like here https://github.com/supercharge/redis-github-action/blob/main/README.md#using-a-custom-redis-image

Reply:
done